stable/mongodb/README.md

```diff
@@ -206,3 +211,14 @@ Use the workaround below to upgrade from versions previous to 5.0.0. The followi
 ```consoloe
 $ kubectl delete statefulset my-release-mongodb-arbiter my-release-mongodb-primary my-release-mongodb-secondary --cascade=false
 ```
+
+## Configure Ingress
+Currently we can expose mongodb externally using nginx-ingres. If you are using `nginx-ingress=1.3.x` check [this options](https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L383)
+on how to expose TCP ports. If you installed mongodb in `default` namespace, then an example would be:
+```yaml
+tcp:
+  27017: "default/mongodb:27017"
+```
```

Comment:
This looks awkward to me. This chart (MongoDB) should be creating the corresponding ConfigMap instead of using the NGINX Ingress Chart to create a configuration object which is meant to other chart.

Reply:
Well, I was testing it with `nginx-ingress`. Basically, if you already have `nginx-ingress` running and already created a ConfigMap for that, then we will have conflict trying to create our own ConfigMap. I see your point and yes would be great to have such a config map. So, this feature is specific for `nginx-ingress` for now.

Reply:
It's okay if this feature is specific for `nginx-ingress` as long as it's indicated in the README.md. Could you please include the corresponding ConfigMap in the PR?

Reply:
Well, still is confusing for me why we need ConfigMap then? If we check https://kubernetes.github.io/ingress-nginx/user-guide/exposing-tcp-udp-services/ then we see that nginx is started with `--tcp-services-configmap` and here we give the name of the config map. To use ingress for MongoDB basically, you have to use `nginx-ingress`.

Reply:
It's a matter of design. 

When I install the MongoDB chart, I expect it to create all the resources it needs to work. For instance, we are creating an Ingress object despite it's going to be consumed by the Ingress Controller. Why? Because this Ingress object is exclusively tied to the MongoDB I'm deploying.

This ConfigMap is the same. It's not going to be consumed by MongoDB but the Ingress Controller instead. 

Maybe I'm understand it incorrectly, is this ConfigMap going to be reused for mapping every port? Even those not related with MongoDB? In that case, I'm okay with leaving it outside the chart since it's not exclusively tied to MongoDB.

Reply:
Yes, it's not related to MongoDB only, all the other ports: redis, postgres are added in the same ConfigMap.

Reply:
Oh... I see.. In that case I agree with your approach but let's try to highlight this requirement. I'd use sth like:

~~~
## Ingress Configuration

MongoDB can exposed externally using the [NGINX Ingress Controller](https://github.com/kubernetes/ingress-nginx). To do so, it's necessary to:

- Install the MongoDB chart setting the parameter `ingress.enabled=true`.
- Create a ConfigMap to map the external port to use and the internal service/port where to redirect the requests (see https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/exposing-tcp-udp-services.md for more information). 

For instance, if you installed the MongoDB chart in the `default` namespace, you can install the [stable/nginx-ingress chart](https://github.com/helm/charts/tree/master/stable/nginx-ingress) setting the "tcp" parameter in the **values.yaml** used to install the chart as shown below:

```yaml
...

tcp:
  27017: "default/mongodb:27017"
```
~~~

Reply:
Seem good, updated that.